diff --git a/lib/typescript.js b/lib/typescript.js
index c2688dd27c33549cd6ebcddae79077fe379bf7f6..cb04daaaa1cf0e9fb38c3b388cc3631453cb2f2c 100644
--- a/lib/typescript.js
+++ b/lib/typescript.js
@@ -20836,6 +20836,10 @@ var ts;
         var includeFileRegexes = patterns.includeFilePatterns && patterns.includeFilePatterns.map(function (pattern) { return getRegexFromPattern(pattern, useCaseSensitiveFileNames); });
         var includeDirectoryRegex = patterns.includeDirectoryPattern && getRegexFromPattern(patterns.includeDirectoryPattern, useCaseSensitiveFileNames);
         var excludeRegex = patterns.excludePattern && getRegexFromPattern(patterns.excludePattern, useCaseSensitiveFileNames);
+
+        console.log({
+            includeFileRegexes
+        });
         // Associate an array of results with each include regex. This keeps results in order of the "include" order.
         // If there are no "includes", then just put everything in results[0].
         var results = includeFileRegexes ? includeFileRegexes.map(function () { return []; }) : [[]];