diff --git a/source/package-json.d.ts b/source/package-json.d.ts
index 599155381372f21c64b9110a02ec6f1b3088c3cd..78a5555e2fd5d003964277f13c0db3c3e0a71932 100644
--- a/source/package-json.d.ts
+++ b/source/package-json.d.ts
@@ -208,27 +208,12 @@ declare namespace PackageJson {
 	*/
 	export type Dependency = Partial<Record<string, string>>;
 
-	/**
-	Conditions which provide a way to resolve a package entry point based on the environment.
-	*/
-	export type ExportCondition = LiteralUnion<
-	| 'import'
-	| 'require'
-	| 'node'
-	| 'node-addons'
-	| 'deno'
-	| 'browser'
-	| 'electron'
-	| 'react-native'
-	| 'default',
-	string
-	>;
-
 	/**
 	A mapping of conditions and the paths to which they resolve.
 	*/
-	type ExportConditions = {[condition in ExportCondition]?: Exports};
-
+	type ExportConditions = { // eslint-disable-line @typescript-eslint/consistent-indexed-object-style
+		[condition: string]: Exports;
+	};
 	/**
 	Entry points of a module, optionally with conditions and subpath exports.
 	*/